templates/components/itilobject/fields_panel.html.twig

```diff
@@ -167,7 +170,10 @@
                ) }}
             {% endif %}
 
-            {% if item.isNewItem() and item.getType() == 'Ticket' %}
+            {% if item.isNewItem() and item.getType() == 'Ticket' and call('countElementsInTable', [
+               'Contract'|itemtype_table,
+               {'is_deleted': 0}
+            ]) > 0 %}
```

Comment:
Strating to intoducing direct call to DB in views seems a bad idea to me. I will try to propose an alternative solution that will consist in adding a `hide_if_no_elements` option to the dropdown method.